test/acceptance/create_webdriver.robot

```diff
@@ -29,6 +29,40 @@ Create Webdriver With Bad Keyword Argument Dictionary
     Run Keyword And Expect Error    kwargs must be a dictionary.
     ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
 
+Get Browser Capabilities Using Dictionary
+    [Documentation]    Gets current browser capabilities from WebDriver
+    [Setup]    Set Driver Variables
+    Create Webdriver    ${DRIVER_NAME}
+    Go To    ${FRONT PAGE}    # If we don't do this then no version details
+    ${capabilities}=    Get Browser Capabilities
+    Log    Capabilities = ${capabilities}
+    Log    Browser is ${capabilities['browserName']}, version ${capabilities['version']}
+    ${browser}=    Normalize Browser Name
+    Should Match    ${capabilities['browserName']}    ${browser}
+    Run Keyword If    "${browser}" != "internet explorer"    Log    Rotatable = ${capabilities['rotatable']}
+    # Disabled next step to PASS in RF pre 2.9
+    # Run Keyword If    "${browser}" == "internet explorer"    Run Keyword And Expect Error    *    Log
+    #     ...    Rotatable = ${capabilities['rotatable']}
```

Comment:
Before RF 2.9 it fails with exception in code before reaching the error. I though it is useful info to leave there.